### ISSUE TITLE
Install spinnaker sdk full and python wrapper in Mac

### ISSUE THREAD

**Issue by spsaswat** [spsaswat/APPF_FLIR#10]
Install spinnaker SDK full and python wrapper in Mac

**Comment by zzzshuang** [comment]
Hi Saswat @spsaswat, I encountered an issue where the Spinnaker SDK only supports Intel chip Macs, but I'm using a Mac with an M chip, so I can't install the package. Following your previous suggestion(using TeamViewer), I attempted to use ANU Virtual Desktop to install TeamViewer and Spinnaker. Unfortunately, it seems that the ANU virtual desktop doesn't permit the downloading and installation of software. However, I discovered that Parallels Desktop allows me to install Spinnaker, so I am currently attempting to see if I can successfully connect using Parallels Desktop.
